action.yml

```diff
@@ -1,4 +1,4 @@
-name: 'Sync Configuration'
+name: '"Azure App Configuration Sync'

```

Comment:
Extra quote at the beginning.

Reply:
Sent separate PR to remove the extra quote